SleepNumberController_App.groovy

```diff
@@ -190,82 +248,132 @@ def installed() {
 def updated() {
   unsubscribe()
   unschedule()
+  app.removeSetting('hubitatQueryString')
+  app.removeSetting('requestType')
+  app.removeSetting('requestPath')
+  app.removeSetting('requestBody')
+  app.removeSetting('requestQuery')
+  state.remove('variableRefresh')
   state.session = null // next run will refresh all tokens/cookies
-  state.variableRefresh = ""
   initialize()
-  if (enableDebugLogging) {
-    runIn(1800, logsOff)
+  if ((Boolean)settings.enableDebugLogging) {
+    runIn(7200L, logsOff)
```

Comment:
Can you change this back to 1800?  It matches the text shown to the user that way.

Reply:
yes, that can be put back or rejected.  Was there for testings